pyproject.toml

```
@@ -46,7 +46,7 @@ zstd = [
   "zstandard>=0.18.0",
 ]
 secure = [
-  "pyOpenSSL>=0.14",
+  "pyOpenSSL>=17.1.0",
   "cryptography>=1.3.4",
```

What do you think about making this in line with what [pyOpenSSL requires](https://github.com/pyca/pyopenssl/blob/17.1.0/setup.py#L99) to prevent any resolving issues? 

```suggestion
  "cryptography>=1.9",
```

---

Thanks, good idea. We will have to update the changelog too